zerver/openapi/zulip.yaml

```diff
@@ -15689,6 +15689,20 @@ paths:
                                   nullable: true
                                 is_announcement_only: {}
                                 can_remove_subscribers_group: {}
+                                stream_weekly_traffic:
+                                  type: integer
+                                  nullable: true
+                                  description: |
+                                    The average number of messages sent to the stream per week, as
+                                    estimated based on recent weeks, rounded to the nearest integer.
+
+                                    If `null`, no information is provided on the average traffic.
+                                    This can be because the stream was recently created and there
+                                    is insufficient data to make an estimate, or because the server
+                                    wishes to omit this information for this client, this realm, or
+                                    this endpoint or type of event.
+
+                                    **Changes**: New in Zulip 8.0 (feature level 199).
```

Comment:
This could be extended to say "Previously, these data were only available in subscription objects."